install.md

```diff
@@ -0,0 +1,100 @@
+
+## Pre-requisites
+
+You must have a the ability to run root containers in a cluster
+with at least 12 vCPUs and 45G of memory and a minumum of three nodes.
+You will also need to provide a docker compatable registry. 
+
+## Configure your Registry
+In order to make this install simple to walk through we recomend you 
+store your docker registry details in a an environment variable. This
+install guide uses gcr on gke. 
+
+```
+export KF_REGISTRY=gcr.io/<PROJECT_ID>
+```
+
+
+## Install Istio && Knative
+
+Install istio CRDs and deploy pods and label the default namespace. 
+```
+kubectl apply --filename https://github.com/knative/serving/releases/download/v0.5.0/istio-crds.yaml && \
+kubectl apply --filename https://github.com/knative/serving/releases/download/v0.5.0/istio.yaml && \
+kubectl label namespace default istio-injection=enabled
+```
+
+Install knative CRDs.
+```
+kubectl apply --selector knative.dev/crd-install=true \
+--filename https://github.com/knative/serving/releases/download/v0.5.0/serving.yaml \
+--filename https://github.com/knative/build/releases/download/v0.5.0/build.yaml \
+--filename https://github.com/knative/eventing/releases/download/v0.5.0/release.yaml \
+--filename https://github.com/knative/eventing-sources/releases/download/v0.5.0/eventing-sources.yaml \
+--filename https://github.com/knative/serving/releases/download/v0.5.0/monitoring.yaml \ &&
+--filename https://raw.githubusercontent.com/knative/serving/v0.5.0/third_party/config/build/clusterrole.yaml 
+```
+
+Install Knative PODs
+```
+kubectl apply --filename https://github.com/knative/serving/releases/download/v0.5.0/serving.yaml \
+--filename https://github.com/knative/build/releases/download/v0.5.0/build.yaml \
+--filename https://github.com/knative/eventing/releases/download/v0.5.0/release.yaml \
+--filename https://github.com/knative/eventing-sources/releases/download/v0.5.0/eventing-sources.yaml \
+--filename https://github.com/knative/serving/releases/download/v0.5.0/monitoring.yaml \
+--filename https://raw.githubusercontent.com/knative/serving/v0.5.0/third_party/config/build/clusterrole.yaml
+```
+
+
+## Upload buildpacks
+Buildpacks are provided by the operator and can be uploaded to Knative using the
+CLI. A set of buidpacks is included in this repo. Change into the `buildpack-samples`
+directory run the following command. 
+
+```
+kf upload-buildpacks --container-registry $KF_REGISTRY
+```
+
+## Push your first app
+At this point you are ready to deploy your first app using `kf`. Run the following command 
+to push your first app. 
+
+```
+kf push helloworld --container-registry $KF_REGISTRY
+```
+
+## Install the service catalog
```

Comment:
Should we link to https://kubernetes.io/docs/tasks/service-catalog/install-service-catalog-using-helm/ ?

Reply:
My goal on `install.md` is all the commands you need in one place. This started as a scratch file with all the commands I needed originally. Linking to external instructions increases the likelihood someone churns on the install process.

Reply:
What about including the link as well.

Reply:
I like the idea of both capturing the commands as well as linking to the source.

Reply:
Added some overall context to the guide a link over to knative.